lerobot/common/robot_devices/robots/lekiwi_remote.py

```diff
@@ -0,0 +1,197 @@
+import json, time, threading, cv2, base64, zmq
+from pathlib import Path
+from lerobot.common.robot_devices.robots.mobile_manipulator import LeKiwi
+
+def setup_zmq_sockets(config):
```

Comment:
You need to run:
```bash
pre-commit install
pre-commit run --all-files
```

Reply:
<img width="844" alt="Screenshot 2025-02-24 at 10 11 35" src="https://github.com/user-attachments/assets/b4c89252-1e0a-4a25-8ed7-06e2e8d670cc" />